doc/Learn/Markup/Appendix/AccessingControlInstance.md

```diff
@@ -4,14 +4,14 @@ uid: Uno.Extensions.Markup.Appendix.AccessingControlInstance
 
 # Accessing a Control Instance
 
-There are a number of times where you may find yourself wanting to access the instance of a control that you initialized. This could be useful for bindings or even to later configure Event Handlers on the control. You can most easily do this with the `Assign` method as follows:
+There are a number of times where you may find yourself wanting to access the instance of a control that you initialized. This could be useful for bindings or even to later configure Event Handlers on the control. You can most easily do this with the `Name` method as follows:
```

Comment:
```suggestion
There are a number of times when you may find yourself wanting to access the instance of a control that you initialized. This could be useful for bindings or even to later configure Event Handlers on the control. You can most easily do this with the `Name` method as follows:
```